changed.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+const eslintConfig = require('./.eslintrc');
+
+// Add the config for preventing deprecated component-library imports
+eslintConfig.plugins = [...eslintConfig.plugins, 'deprecate'];
+eslintConfig.rules = {
+  ...eslintConfig.rules,
+  'deprecate/import': [
+    'error',
+    {
+      name:
+        '@department-of-veterans-affairs/component-library/CollapsiblePanel',
+      use: '<va-accordion>',
+    },
+  ],
+};
```

Comment:
Add in the `deprecate/import` rule and configure it to error out on `CollapsiblePanel` imports.